prompts/configs.go

```diff
@@ -308,19 +310,26 @@ func getValuesForField(
 	if field.Description != nil {
 		description = *field.Description
 	}
-	if field.Name == "packageName" && isQuickstart && sdkClassName != "" {
+	if field.Name == "packageName" && isQuickstart {
+		// By default we base the package name on the SDK class name
+		packageName := sdkClassName
+
+		if quickstart.IsUsingTemplate && quickstart.Defaults.TemplateData != nil {
+			packageName = quickstart.Defaults.TemplateData.PackageName
```

Comment:
Is packageName never empty/default? Do we want to prefer sdkClassName if they didn't choose a specific packageName? Maybe not

Reply:
No i dont think so - we do prefer it currently - this is just the case of using a template.